src/Compilers/CSharp/Portable/Binder/ExpressionVariableFinder.cs

```diff
@@ -348,6 +339,50 @@ public override void VisitBinaryExpression(BinaryExpressionSyntax node)
             operands.Free();
         }
 
+        public override void VisitInvocationExpression(InvocationExpressionSyntax node)
+        {
+            if (receiverIsInvocation(node, out InvocationExpressionSyntax nested))
+            {
+                var invocations = ArrayBuilder<InvocationExpressionSyntax>.GetInstance();
+
+                invocations.Push(node);
+
+                node = nested;
+                if (receiverIsInvocation(node, out nested))
```

Comment:
It looks like we'll push at most two nodes into the `invocations` queue. Should this be a `while` loop instead?

Reply:
> Should this be a while loop instead?

Yes, this is a typo.